### PR TITLE
feat - Added snackbar Component

### PR DESCRIPTION
**Pull request by lannisterrr**
https://user-images.githubusercontent.com/29068538/152639105-0e594277-7789-4f33-9481-a56bd5ab2fe3.mp4

Added SnackBar Component with Primary Color as Background.
Added Close Button in the snackbar.

### CODE REVIEW

File: components/snackbar/snackbar.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  <body>
+    <div class="centered-row space-around-row component-border p-4">
+      <button class="btn btn-custom m-h-3 t-c-1 snackbar-button">
+        show snackbar
+      </button>
+    </div>
+    <div class="snack-bar-container p-3">
+      <div class="snack-bar">
+        <p class="f-5">Can't send photo. Retry in 5 seconds.</p>
+        <a class="f-bold t-c-1 f-5 p-3" href="#">Retry</a>
```
Comment (AnuragSahu11) [inline components/snackbar/snackbar.html:23]: t-c-1 is not very descriptive about what it does.

**Review by AnuragSahu11 — COMMENTED** [review]
looks good

**Review by ashishnnnnn — APPROVED** [review]
Code seems good to merge

File: components/snackbar/snackbar.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  </head>
+  <body>
+    <div class="centered-row space-around-row component-border p-4">
+      <button class="btn btn-custom m-h-3 t-c-1 snackbar-button">
+        show snackbar
+      </button>
+    </div>
+    <div class="snack-bar-container p-3">
+      <div class="snack-bar">
+        <p class="f-5">Can't send photo. Retry in 5 seconds.</p>
```
Comment (bajajcodes) [inline components/snackbar/snackbar.html:22]: Selectors like `f-5`, `m-h-3` and such if are from cdn then no issues, else if possible give self-describing selector names.

**Review by bajajcodes — COMMENTED** [review]
LGTM , except selectors name used in snackbar.html.

File: components/snackbar/snackbar.html (hunk 2 above)
Comment (lannisterrr) [reply components/snackbar/snackbar.html:22]: Hey they are are part of my design system , and I have been using these all over my component library.

File: components/snackbar/snackbar.html (hunk 1 above)
Comment (lannisterrr) [reply components/snackbar/snackbar.html:23]: text color 1 . I am using this format all over my code base to make my workflow faster.

**Comment by bajajcodes** [comment]
Good 😊 would love to read your documentation.

**Pull request merged**